tests/publiccloud/prepare_tools.pm

```diff
@@ -46,9 +46,15 @@ sub run {
         assert_script_run("pip3 install -q keyring");
     }
     elsif (is_sle) {
-        zypper_call('rr Cloud_Tools');
-        zypper_call('ref');
         zypper_call('-q in aws-cli');
+
+        # boo#1122199
```

Comment:
I recommend to record a soft-failure for this one

Reply:
this PR actually fixes the problem. Shall we remove the bug report in the comment?

Reply:
and I would recommend to remove the comment. Bug has resolved state and actually was not a bug from my POV

Reply:
Ok, I will add  it.

Reply:
Now I am also confused. We should use soft-fails *only* for bugs that are open and we want to be fixed. This looks pretty messy now. To my understanding we should use the tools from the offical SLE repos. If we can't then the OBS repos should provide the right dependencies. Maybe you could try to add a multibuild package self-test to aws-cli like I did for azure-cli already in https://build.opensuse.org/request/show/651347 ?

Reply:
ok